terraform/deployments/monitoring-test/base/main.tf

```diff
@@ -0,0 +1,71 @@
+terraform {
+  backend "s3" {
+    bucket  = "govuk-terraform-test"
+    key     = "projects/monitoring.tfstate"
+    region  = "eu-west-1"
+    encrypt = true
+  }
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.13"
+    }
+  }
+}
+
+provider "aws" {
+  region = "eu-west-1"
+
+  assume_role {
+    role_arn = var.assume_role_arn
+  }
+}
+
+data "terraform_remote_state" "infra_networking" {
+  backend = "s3"
+  config = {
+    bucket   = var.govuk_aws_state_bucket
+    key      = "govuk/infra-networking.tfstate"
+    region   = "eu-west-1"
+    role_arn = var.assume_role_arn
+  }
+}
+
+data "terraform_remote_state" "infra_security_groups" {
+  backend = "s3"
+  config = {
+    bucket   = var.govuk_aws_state_bucket
+    key      = "govuk/infra-security-groups.tfstate"
+    region   = "eu-west-1"
+    role_arn = var.assume_role_arn
+  }
+}
+
+data "terraform_remote_state" "govuk" {
+  backend = "s3"
+  config = {
+    bucket   = "govuk-terraform-test"
+    key      = "projects/govuk.tfstate"
+    region   = "eu-west-1"
+    role_arn = var.assume_role_arn
+  }
+}
+
+data "aws_iam_role" "execution" {
+  name = "fargate_execution_role"
```

Comment:
Should we pull this out of remote state too? Same for terraform/deployments/monitoring-test/grafana/app-config/main.tf - would be nice to pull data out consistently. Feel free to add TODOs for these and we could change them later.

Reply:
yep, I can try to do that if easier.

Reply:
Your call. Feel free to use TODOs.

Reply:
I created some iams directly for monitoring so that it is not dependent on govuk.